### ISSUE TITLE
Max Enemies not read as Friends

### ISSUE THREAD

**Issue by SampeyFS** [TheGameCreators/GameGuruRepo#5458]
![Screenshot 2024-04-14 123001](https://github.com/TheGameCreators/GameGuruRepo/assets/108415264/376d3aa7-0f5d-40f2-a3a8-75c5392314bf)

![image](https://github.com/TheGameCreators/GameGuruRepo/assets/108415264/df0f36af-61f0-4f7c-ab3e-6ba2c5407314)

![image](https://github.com/TheGameCreators/GameGuruRepo/assets/108415264/19c88e85-d18d-42de-96a5-73e4f61b2618)

the various npcs in the project are set to allies instead max reads them as enemies

**Comment by SampeyFS** [comment]
Friends get tagged as enemies why?

**Comment by Kitakazi** [comment]
@LeeBamberTGC I think he means on the map like the RPG Hud Map, its showing Allies as being colored red which is what the enemies are colored.

**Comment by SampeyFS** [comment]
Exactly, everything is the same, allies and enemies, a minestrone is like that

**Comment by LeeBamberTGC** [comment]
@SampeyFS Understood. At the mo, the red dots only show characters, and do not distinguish between different kinds of characters. I have changed this to a feature request. Right now there are only four types of dot:
![image](https://github.com/TheGameCreators/GameGuruRepo/assets/35840583/cf0556a7-867c-481b-bffe-029326238513)

**Comment by Necrym59** [comment]
Enhancment Details now in 
https://github.com/TheGameCreators/GameGuruRepo/issues/5481
